src/Data/Fin/Permutation.agda

```diff
@@ -221,6 +248,35 @@ insert {m} {n} i j π = permutation to from inverseˡ′ inverseʳ′
     punchIn j (punchOut j≢k)                                               ≡⟨ punchIn-punchOut j≢k ⟩
     k                                                                      ∎
 
+------------------------------------------------------------------------
+-- Swapping
+
+-- Takes a permutation m → n and creates a permutation
+-- suc (suc m) → suc (suc n) by mapping 0 to 1 and 1 to 0 and
+-- then applying the input permutation to everything else
+swap : Permutation m n → Permutation (suc (suc m)) (suc (suc n))
+swap {m} {n} π = permutation to from inverseˡ′ inverseʳ′
```

Comment:
Is this definition easier to describe/define indirectly, via the composition of
* `eval ((0F , 1F) ∷ (1F , 0F))` using `Data.Fin.Permutation.Transposition.List`
* `lift₀ {suc m} {suc n} (lift₀ {m} {n} π)`

Reply:
Yup, much nicer! Good spot!